pykern/pkcli/github.py

```diff
@@ -427,7 +427,7 @@ def _repo(self, repo):
 
         def _clone(suffix):
             base = bd + suffix
-            prev = self._prev_backup(base, ext="*")
+            prev = self._prev_backup(base, ext="")
```

Comment:
isn't it needed for wiki?

Reply:
I don't believe so. In the wiki case we'll look for 'radiasoft-<repo>.wiki.git' which will work. Rigth?